main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+long long factorial (int n)
+{
+	if(n==0 || n==1)
```

Comment:
For better readability add spaces after if,   add spaces after and before ==, *, -,...

Reply:
Thank you for review. Code is changed, committed.